frontend/appflowy_flutter/lib/plugins/database_view/calendar/presentation/calendar_day.dart

```diff
@@ -143,11 +148,20 @@ class _DayEventCell extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+    final borderSide = BorderSide(
+      color: Theme.of(context).dividerColor,
+      width: 1.0,
+    );
+
     return FlowyHover(
       child: GestureDetector(
         onTap: onClick,
         child: Container(
           padding: const EdgeInsets.symmetric(horizontal: 8),
+          decoration: BoxDecoration(
+            border: Border.fromBorderSide(borderSide),
```

Comment:
Doesn't really make a difference to declare a variable if you're just going to use it once.